config/envs/default.py

```diff
@@ -186,3 +186,6 @@ class DefaultConfig:
     REDIS_INSTANCE_URI = getenv("REDIS_INSTANCE_URI", "redis://localhost:6379")
     TOGGLES_URL = REDIS_INSTANCE_URI + "/0"
     FEATURE_CONFIG = CommonConfig.dev_feature_configuration
+
+    # LRU cache settings
+    LRU_CACHE_TIME = 300  # in seconds
```

Comment:
Add this setting to uat/production environment to be 3600 - 5 minutes on dev is fine, 1 hour on uat and production makes sense